lib/gps.js

```diff
@@ -308,6 +397,47 @@ GPS.prototype.sendCommand = function(string) {
   this.io.serialWrite(state.portId, cc);
 };
 
+/*
+ * SkyTraq Binary Message Protocol
+ * http://cdn.sparkfun.com/datasheets/Sensors/GPS/Venus/638/doc/AN0003_v1.4.19.pdf
+ */
+GPS.prototype.sendSkyTraqCommand = function(payload, callback) {
```

Comment:
Is this specific to _just_ this model? There shouldn't be component specific methods like this, so I think you and @dtex should coordinate on this implementation.